app/components/Review/ProjectFundingTable.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+  formatRow,
+  StyledTable,
+  StyledColLeft,
+  StyledColRight,
+  StyledH4,
+  StyledTitleRow,
+} from './Table';
+
+import formatMoney from '../../utils/formatMoney';
+
+const ProjectFundingTable = ({ formData, subschema }: any) => {
+  const rows = Object.keys(subschema.properties);
+
+  return (
+    <StyledTable>
+      {rows.map((row, i) => {
+        const title = subschema.properties[row].title;
+        const value = formData ? formatRow(formData[row]) : ' ';
+
+        return (
+          <tbody key={row}>
+            {i === 0 && (
+              <tr>
+                <StyledTitleRow colSpan={2}>
+                  <StyledH4>Amount requested under CCBC</StyledH4>
+                </StyledTitleRow>
+              </tr>
+            )}
+            {i === 6 && (
+              <tr>
+                <StyledTitleRow colSpan={2}>
+                  <StyledH4>Amount applicant will contribute</StyledH4>
+                </StyledTitleRow>
+              </tr>
+            )}
+            {i === 12 && (
+              <tr>
+                <StyledTitleRow colSpan={2}>
+                  <StyledH4>
+                    Amount requested under Canadian Infrastructure Bank
+                  </StyledH4>
+                </StyledTitleRow>
+              </tr>
+            )}
+            <tr key={i}>
```

Comment:
There wasn't a good key and I was under the impression that i is fine in react if you're dealing with static data and not reordering items.

Reply:
Yeah since these are static the `avoid index as key` here doesn't necessarily need to be followed. It is better if we can get some unique keys though but not required